examples/ex5.cpp

```diff
@@ -264,7 +264,16 @@ int main(int argc, char *argv[])
    visit_dc.RegisterField("pressure", &p);
    visit_dc.Save();
 
-   // 14. Send the solution by socket to a GLVis server.
+   // 14. Save data in the Paraview format
+   ParaviewDataCollection paraview_dc("PExample5", mesh);
+   paraview_dc.SetLevelsOfDetail(2);
+   paraview_dc.SetCycle(1);
+   paraview_dc.SetTime(0.0); //set the time
+   paraview_dc.RegisterField("velocity",&u);
+   paraview_dc.RegisterField("pressure",&p);
+   paraview_dc.Save();
+
+   // 15. Send the solution by socket to a GLVis server.
```

Comment:
Should we make the VisIt and/or Paraview output optional?

Reply:
I can do it optional - I am not sure what would be better. Any input is welcome :-)